src/main/scala/io/iohk/ethereum/vm/EvmConfig.scala

```diff
@@ -175,7 +175,7 @@ object FeeSchedule {
   }
 
   class HomesteadFeeSchedule extends FrontierFeeSchedule {
-    override val G_txcreate = 53000
+    override val G_txcreate = 32000
```

Comment:
so what happens with 53000? parity seems to keep them in separate config entries: create_gas = 32000 and tx_create_gas which is 53000.